engine/src/main/java/org/terasology/config/flexible/typehandling/LocaleTypeHandler.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.config.flexible.typehandling;
+
+import org.terasology.persistence.typeHandling.StringRepresentationTypeHandler;
+
+import java.util.Locale;
+
+/**
+ * Locale type handler for {@link org.terasology.config.SystemConfig#locale}
+ */
+public class LocaleTypeHandler extends StringRepresentationTypeHandler<Locale> {
```

Comment:
Similar to above -- since `Locale` is a SDK class, it might be better to move this to a subpackage in `org.terasology.persistence.typeHandling`.

Reply:
`* Serializes objects of type {@link java.util.Locale}` --  no need to reference the particular use case for which this type handler was created.